docs/Middleware.md

```diff
@@ -38,6 +38,8 @@ Fastify middleware also do not expose the `send` method or other methods specifi
 #### Restrict middleware execution to a certain path(s)
 If you need to run a middleware only under certain path(s), just pass the path as first parameter to `use` and you are done!
 
+The middleware syntax is different from the fastify route syntax. We use [middie](https://github.com/fastify/middie) to support express.js.
+
 *Note that this does not support routes with parameters, (eg: `/user/:id/comments`) and wildcards are not supported in multiple paths.*
```

Comment:
The new sentence could be included in this note?
It is referring to the wildcards